primitives/src/pcs/multilinear_kzg/mod.rs

```diff
@@ -25,40 +25,48 @@ use ark_ff::PrimeField;
 use ark_poly::{DenseMultilinearExtension, MultilinearExtension};
 use ark_serialize::{CanonicalDeserialize, CanonicalSerialize, Read, SerializationError, Write};
 use ark_std::{
-    end_timer, format, marker::PhantomData, rand::RngCore, rc::Rc, start_timer, string::ToString,
-    vec, vec::Vec, One, Zero,
+    borrow::Borrow,
+    end_timer, format,
+    marker::PhantomData,
+    rand::{CryptoRng, RngCore},
+    rc::Rc,
+    start_timer,
+    string::ToString,
+    vec,
+    vec::Vec,
+    One, Zero,
 };
 use batching::{batch_verify_internal, multi_open_internal};
 use srs::{MultilinearProverParam, MultilinearUniversalParams, MultilinearVerifierParam};
 use util::merge_polynomials;
 
 /// KZG Polynomial Commitment Scheme on multilinear polynomials.
-pub struct KZGMultilinearPCS<E: PairingEngine> {
+pub struct MultilinearKZGPCS<E: PairingEngine> {
```

Comment:
sorry to be picky, what about camel-case some abbreviation when there's multiple of them?

`MultilinearKZGPCS` -> `MultilinearKzgPCS` or `MultilinearKzgPcs` (I prefer the first.
(similar comment to other names)